upload/views/bundle_analysis.py

```diff
@@ -150,9 +154,17 @@ def post(self, request: HttpRequest) -> Response:
                 request=self.request,
                 repository=repo,
                 is_shelter_request=self.is_shelter_request(),
+                position="end",
             ),
         )
 
+        dispatch_upload_task(
```

Comment:
why move this?

Reply:
Just wanted the metric to come before the actual dispatching